must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -67,12 +66,14 @@ oc_yamls+=("hpa")
 oc_yamls+=("subscription")
 oc_yamls+=("installplan")
 
-echo "collecting dump of namespace" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+echo "collecting existing namespaces" | tee -a "${BASE_COLLECTION_PATH}"/gather-debug.log
+INSTALL_NAMESPACE=$(oc get storagecluster -A --no-headers | awk '{print $1}')
```

Comment:
This can return multiple storageclusters. We should iterate through each one.

Reply:
As per discussion with @jilju he mentioned there will be only one namespace, hence removed the iteration. Can you confirm @jilju